### PR TITLE
add missing sense key and sense code qualifier strings

### PR DESCRIPTION
**Pull request by ttlajus**
Co-authored-by: Marcus Paradies <marcus.paradies@gmail.com>

### CODE REVIEW

File: lib/zbc.c [hunk 1]
```diff
@@ -38,10 +38,12 @@ static struct zbc_sg_sk_s {
 	enum zbc_sk	sk;
 	const char	*sk_name;
 } zbc_sg_sk_list[] = {
+	{ ZBC_SK_NOT_READY,			"Not-ready"			},
```
Comment (damien-lemoal) [inline lib/zbc.c:41]: Can you fix the indentation so that everything aligns nicely ?

File: lib/zbc.c [hunk 2]
```diff
@@ -38,10 +38,12 @@ static struct zbc_sg_sk_s {
...
+	{ ZBC_SK_MEDIUM_ERROR,		"Medium-error"		},
 	{ ZBC_SK_ILLEGAL_REQUEST,	"Illegal-request"	},
 	{ ZBC_SK_DATA_PROTECT,		"Data-protect"		},
 	{ ZBC_SK_ABORTED_COMMAND,	"Aborted-command"	},
-	{ 0,				NULL }
+	{ 0,						NULL				}
```
Comment (damien-lemoal) [inline lib/zbc.c:46]: Same here

**Comment by damien-lemoal** [comment]
Also please add a commit message (a simple one is OK) and your Signed-off-by tag.

File: lib/zbc.c (hunk 1 above)
Comment (ttlajus) [reply lib/zbc.c:41]: @damien-lemoal The two lines you pointed out actually show up nicely in both of my text editors. I imagine this has to do with how Github renders tabs. If I "fix" the indentation to make it look right on Github, it won't look right in text editors. What do you want?

Comment (damien-lemoal) [reply lib/zbc.c:41]: Here is what I get when I apply your patch:
```
/**
 * Sense key strings.
 */
static struct zbc_sg_sk_s {
        enum zbc_sk     sk;
        const char      *sk_name;
} zbc_sg_sk_list[] = {
        { ZBC_SK_NOT_READY,                     "Not-ready"                     },
        { ZBC_SK_MEDIUM_ERROR,          "Medium-error"          },
        { ZBC_SK_ILLEGAL_REQUEST,       "Illegal-request"       },
        { ZBC_SK_DATA_PROTECT,          "Data-protect"          },
        { ZBC_SK_ABORTED_COMMAND,       "Aborted-command"       },
        { 0,                                            NULL                            }
};
```

So definitely not aligned. Check with a simple plain text editor like vim please.
Anyway, that is fine, I can fix that when applying. Please update your PR with a proper commit message (not just a title) and add your Signed-off-by tag.

Comment (ttlajus) [reply lib/zbc.c]: @damien-lemoal My point was simply that these two lines will always be misaligned for some people. Anyways, I've modified the commit per your instructions.

**Review by damien-lemoal — COMMENTED** [review]
The commit message is still missing a boby... It should be something like:

```
lib: add missing sense key and sense code qualifier strings

Add "not ready" and "medium error" sense keys and ... Blah.

Signed-off-by: ttlajus <40629580+ttlajus@users.noreply.github.com>
Co-authored-by: Marcus Paradies <marcus.paradies@gmail.com>
```

And please add a proper SoB tag. Please use your name and email address, similarly to the Co-authored-by tag you have.

**Comment by ttlajus** [comment]
@damien-lemoal Alright at this point I think I'll just let you do it. It's a simple patch.

**Comment by damien-lemoal** [comment]
Fixed and pushed.

**Pull request closed without merging**